handlers.go

```diff
@@ -901,7 +901,11 @@ func GetStaticHandler(c echo.Context) error {
 	bot := c.Get("bot").(*OGame)
 
 	newURL := bot.serverURL + c.Request().URL.String()
-	resp, err := http.Get(newURL)
+	req, err := http.NewRequest("HEAD", newURL, nil)
```

Comment:
I don't think you can just remove the Get request and replace it for a "head" request (only).
You are no longer making the real request...

Reply:
You're right, somehow I mixed it up with the GetStaticHEADRequest code. I'll push an updated one.